docs/adrs/0081-deprecating-devseed-scenarios.md

```diff
@@ -0,0 +1,110 @@
+Since this move data is also used with Playwright tests, an added benefit is
+testing the `testharness` data generation can be done programmatically as part
+of the end-to-end testing.
+
+### Current state of documentation related to test data
+
+Our current documentation portal contains references to both `testharness` and
+`devseed` data. If we were to pick one strategy going forward, we would have to
+update our documentation in a number of places.
+
+[=> Search results for devseed](https://transcom.github.io/mymove-docs/search?q=devseed)
+
+There's also documentation related to `make db_dev_e2e_populate` which uses the
+`devseed` data as well. This documentation would need to be updated and possibly
+removed. Currently there is no equivalent of `make db_dev_e2e_populate` for the
+`testharness` data as the creation of move data with `testharness` is entirely
+browser-UI based at this time.
+
+## Considered Alternatives
+
+- _Do nothing_
+- _Deprecate `devseed` scenarios and provide team-wide guidance on exclusively
+  using `testharness` scenarios_
+- _Deprecate and delete `deveseed` scenarios_
+- _Update `devseed` scenarios and use them in tandem with `testharness`
+  scenarios_
+
+## Decision Outcome
+
+> to be decided
+
+<!--
+- Chosen Alternative: _[alternative 1]_
+- _[justification. e.g., only alternative, which meets KO criterion decision driver | which resolves force | ... | comes out best (see below)]_
+ optional -->
+
+## Pros and Cons of the Alternatives <!-- optional -->
+
+### _Do nothing_
+
+- `+` _No extra work involved._
+- `-` _Teams will remain unsure about whether to use `devseed` or `testharness`
+  data._
+- `-` _Teams may duplicate work as `testharness` data is required for Playwright
+  tests._
+
+### _Deprecate `devseed` scenarios and provide team-wide guidance on exclusively using `testharness` scenarios_
+
+- `+` _Teams will be able to focus on data creation using `testharness` data._
+- `+` _Teams will have `testharness` data for Playwright tests that have been
+  used during the development of features._
+- `-` _Teams will need to over-communicate `testharness` data creation over
+  `devseed` data creation._
+- `-` _Using `testharness` data can slow us down because currently it only
+  creates individual moves rather than subscenarios like `devseed` data which
+  can create multiple moves tied to an Epic._
+  - `?` _This will change the way we create move data. With `testharness` data
+    will need to be created manually._
+  - `?` _Teams will be able to create more specific move data faster._
+
+### _Deprecate and delete `deveseed` scenarios_
+
+- `+` _Teams will be able to focus on data creation using `testharness` data._
+- `+` _Teams will have `testharness` data for Playwright tests that have been
+  used during the development of features._
+- `-` _More work involved in implementing the ADR._
+- _[...]_ <!-- numbers of pros and cons can vary -->
+
```

Comment:
This may be the better solution here. If there is nothing that relies on `devseed` existing, then we can get rid of it pretty quickly. The only thing that uses `devseed` data is the ephemeral deployment.

The biggest lifts here are the documentation updates around `devseed`.

Reply:
This would also get rid of certain `make` targets related to populating the database.